packages/vite/src/node/plugins/preAlias.ts

```diff
@@ -26,18 +26,15 @@ export function preAliasPlugin(config: ResolvedConfig): Plugin {
   return {
     name: 'vite:pre-alias',
     async resolveId(id, importer, options) {
+      if (options.scan || id === '@vite/client' || id === '@vite/env') {
+        return null
```

Comment:
we can directly return here
```suggestion
        return
```